### PR TITLE
All alerts close the connection. Unknown alerts are failures. Fixes #472

### LINKED ISSUE #472: Shall we just get rid of warning alerts except for the end_of_data markers?

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -3049,14 +3049,30 @@ safety limit is reached.
...
+indicate orderly closure of the connection (see {{closure-alerts}}).
+Upon receiving a warning-level alert, the TLS implementation SHOULD
+indicate end-of-data to the application and, if appropriate for
+the alert type, send a closure alert in response.
+
+Fatal-level messages are used to indicate abortive closure of the
+connection (See {{error-alerts}}). Upon receiving a fatal-level alert,
+the TLS implementation SHOULD indicate an error to the application and
+MUST NOT allow any further data to be sent or received on the
+connection.  Servers and clients MUST forget keys, and secrets
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: keys, -> keys

**Comment by ekr** [comment]
@davidben @richsalz PTAL

**Comment by richsalz** [comment]
LGTM

**Comment by davidben** [comment]
lgtm

**Pull request merged**